### DISCUSSION TITLE
Encryption before storing?

### DISCUSSION THREAD

**Discussion by DarioPavlic** [filecoin-project/community#500] [General]
Does this mean users should encrypt data in order to keep it private from anyone else? Isn't this too much to expect from regular users?

![image](https://user-images.githubusercontent.com/45923622/160411638-eef0e805-d6f4-403c-a2a5-53779a257438.png)

**Comment by anorth** [comment]
Yes, but end users are probably not using Filecoin directly. They will probably use an _application_ built on Filecoin that manages encryption (and many other things) for them.

**Reply by DarioPavlic** [reply]
oh great! can you suggest me any applications I could use for that? Also where can i check the pricing of Filecoin storage options?
